Orbitersdk/samples/ProjectApollo/src_lm/lm_telecom.cpp

```diff
@@ -2677,6 +2748,42 @@ void LEM_SteerableAnt::DefineAnimations(UINT idx) {
 	lem->SetAnimation(anim_SBandPitch, sband_proc[0]); lem->SetAnimation(anim_SBandYaw, sband_proc[1]);
 }
 
+VECTOR3 LEM_SteerableAnt::pitchYaw2GlobalVector(double pitch, double yaw, MATRIX3 NBSA)
```

Comment:
NBSA is a geometrical constant for this class, doesn't need to be a parameter for this function

Reply:
Removed NBSA parameter from function input